### PR TITLE
Add field corrections to summarizer

### PR DESCRIPTION
**Pull request by retzkek**
Add a configurable module to lookup field corrections from Elasticsearch, supporting multiple field matches (e.g. correct `VOName` based on both `VOName` and `ReportableVOName` fields). Original field values are saved in a new field named `Raw<fieldname>` (e.g. `RawVOName`).

Also adds a `uri` parameter in the `[ElasticSearch]` configuration section to use Elasticsearch servers not on `localhost:9200`, but does not include changes to use this parameter elsewhere in the agent.

### CODE REVIEW

File: src/graccreq/correct/correct.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                return rec
+        key = self._key(rec)
+        if key in self.corrections:
+            rec[self.dest_field] = self.corrections[key]
+        return rec
+
+
+if __name__ == '__main__':
+    import urllib3
+    urllib3.disable_warnings()
```
Comment (djw8605) [inline src/graccreq/correct/correct.py:101]: Curious, why is this here?

**Review by djw8605 — COMMENTED** [review]
It looks like this will run fast, using hashes and all.  Which is always appreciated!

File: src/graccreq/correct/correct.py (hunk 1 above)
Comment (retzkek) [reply src/graccreq/correct/correct.py:101]: It makes an SSL connection without verifying certs, which causes urllib3 (which the elasticsearch library is using by default) to spew out a bunch of warnings. which can drown out the actually useful debugging messages.

**Comment by djw8605** [comment]
There's a conflict with the file config/gracc-request.toml (I added a new attribute in #39).  Could you merge and resolve the conflict.  Should be very simple.

**Comment by retzkek** [comment]
Let's go ahead and merge this - as I start looking into adding error tagging, I find myself wondering if we should refactor the corrections and OIM classes a bit into some sort of common "filter" framework. Error tagging would be a part of that.

Higher priority things at the moment, error tagging can wait. Will track in [GRACC-31](https://jira.opensciencegrid.org/browse/GRACC-31).

**Comment by djw8605** [comment]
LGTM
